### ISSUE TITLE
Check that all the MIME types that are supported in IJulia also work in our notebook implementation

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/julia-vscode#2239]

**Comment by tdierickx** [comment]
Julia 1.6.1
VS Code 1.58.2
Windows 10 Pro

Hi, just watched your JuliaCon2021 video on YouTube. Nice Work! 

I'm guessing my issue here is related. When I run cells in a Notebook within VSCode using the "Julia VS Code Extension" 1.6.1 kernel, (1) I **can** see packages, variables, etc. in the Julia workspace pane that you showed in the video (very nice!) but (2) sadly, just printing basic DataFrame stuff tries rendering output as "text/latex", by default, which appears to not be supported quite yet, or at least very well yet, based on [this](https://github.com/microsoft/vscode/issues/119536) and I have to keep manually changing every output cell's format to "text/plain" or "text/html". 

See here:
<img src="https://user-images.githubusercontent.com/16019221/127430500-d16f861c-b5ef-4523-9e7a-d2c65cb83e15.gif" height="250">

However, when I run these same cells in a Notebook using my "regularly" installed Julia 1.6.1 kernel, then (1) the exact same code cells print out as "text/html" straight away, by default; _however_ (2) then I don't get the nice Julia workspace features available when choosing the "Julia VS Code Extension" kernel.

<img src="https://user-images.githubusercontent.com/16019221/127430777-d5dbf39b-7bc5-4c7c-b2a9-3983c1ae26d7.png" height="250">)

However, in your YouTube video, I saw you were actually using pipe operator syntax to show 10 rows of your DataFrame and they appear to have rendered as "text/plain" for you but for me I get either "text/html" (using "Julia VS Code Extension" kernel), which is ok, even though I actually do like the "text/plain" better or "application/vnd.dataresourcesource+json" (using "regular" Julia kernel), the latter doesn't render anything at all; I must manually change it to "text/plain" or "text/html"

To summarize, for me this is what I'm getting:
| function | using "regular" Julia kernel (1.6.1) | using "Julia VS Code Extension" kernel (1.6.1) |
|:----------|:----|:----|
| Julia workspace? | no | yes
| first(df,5) | renders as text/html | garbled text/latex output
| df \|> take(5) | renders as application/vnd.dataresourcesource+json <br>(but no output actually displays) | renders as text/html

Thanks for all that you do and keep up good work!
Tom
